components/brave_rewards/resources/extension/brave_rewards/manifest.json

```diff
@@ -15,6 +15,7 @@
     "webRequest",
     "chrome://favicon/*",
     "https://www.twitch.tv/*",
+    "https://www.youtube.com/*",
```

Comment:
do we need this? as all YT related stuff should be done in GL and extension just reads it

Reply:
We do need it, as the background page needs to listen for completed `https://www.youtube.com/api/stats/watchtime?*` requests.